index.html

```diff
@@ -37,23 +37,301 @@ <h6>
+                                <p>
+                                    2015
+                                </p>
+                            </li>
+                        </ul>
+                    </div>
+                    <div class="work-content-text">
+                        <p>
+                            A daily selection of privately personalized reads; no accounts or sign-ups required.
+                        </p>
+                    </div>
+                    <ul class="work-tech-used">
+                        <li>
+                            html
+                        </li>
+                        <li>
+                            css
+                        </li>
+                        <li>
+                            javascript
+                        </li>
+                    </ul>
+                    <div class="action-div">
+                        <button class="secondary-btn"><p>See Project</p></button>
+                    </div>
+                </div>
+            </div>
+            <div class="card">
+                <img src="./img/snapshoot-portfolio3.png" alt="Avatar">
+                <div class="work-description">
+                    <div class="work-title">
+                        <h4 >Tonic</h4>
+                        <ul class="work-info">
+                            <li class="client">
+                                <p>
+                                    Canopy
+                                </p>
+                            </li>
+                            <li class="role">
+                                <p>
+                                    Back End Dev
+                                </p>
+                            </li>
+                            <li class="year">
+                                <p>
+                                    2015
+                                </p>
+                            </li>
+                        </ul>
+                    </div>
+                    <div class="work-content-text">
+                        <p>
+                            A daily selection of privately personalized reads; no accounts or sign-ups required.
+                        </p>
+                    </div>
+                    <ul class="work-tech-used">
+                        <li>
+                            html
+                        </li>
+                        <li>
+                            css
+                        </li>
+                        <li>
+                            javascript
+                        </li>
+                    </ul>
+                    <div class="action-div">
+                        <button class="secondary-btn"><p>See Project</p></button>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="aboutme-section">
+            <div class="aboutme-content">
+                <div class="title">
+                    <h1>About <br>Myself</h1>
```

Comment:
- [ ] I noticed that you put your title in the `h1` tag but the whole `html` file should contain only one `h1` and you already have it. Please kindly change this title to another title like `h2, h3,...` 😁😁